src/core/qgsgeometry.h

```diff
@@ -572,6 +578,12 @@ class CORE_EXPORT QgsGeometry
     @param hasZValue 25D type?*/
     void translateVertex( QgsWkbPtr &wkbPtr, double dx, double dy, bool hasZValue );
 
+    /**Rotate a single vertex around Z axis
```

Comment:
comment needs to be fixed, this is not restricted to "rotate" a vertex, but can be used for any affine transformation (could actually also replace the translateVertex method)